src/sonic-yang-models/yang-models/sonic-policer.yang

```diff
@@ -0,0 +1,125 @@
+/* this is sonic py yang model */
```

Comment:
Please remove this file. It is mistakenly added here and it is part of yang-templates